cif_pow.dic

```diff
@@ -11967,6 +11967,32 @@ save_pd_proc_ls.diffractogram_id
 
 save_
 
+save_pd_proc_ls.number_parameters
```

Comment:
It might be good to add a short note on how this definition differs from `_refine_ls.number_parameters`.